srcpkgs/kitty/template

```diff
@@ -1,24 +1,32 @@
 # Template file for 'kitty'
 pkgname=kitty
-version=0.10.1
+version=0.11.1
 revision=1
 python_version=3
-hostmakedepends="ncurses pkg-config python3-devel wayland-devel wayland-protocols"
+hostmakedepends="ncurses pkg-config python3-Sphinx"
 makedepends="gettext-devel glfw-devel harfbuzz-devel libxkbcommon-devel
  python3-devel wayland-devel wayland-protocols"
 depends="kitty-terminfo"
 conf_files="/etc/kitty/kitty.conf"
 short_desc="Modern, hackable, featureful, OpenGL based terminal emulator"
 maintainer="Julio Galvan <juliogalvan@protonmail.com>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/kovidgoyal/kitty"
+homepage="https://sw.kovidgoyal.net/kitty/"
 distfiles="https://github.com/kovidgoyal/kitty/archive/v${version}.tar.gz"
-checksum=ef22208497a76e2f88ebe56c176e4608f049b056252cf1bf122c9c1ec711cfa6
+checksum=3bbc6b5465d424969b16c5ad7f2f67ffbfe33657fdcb443e1bcc11aa00726841
+
+if [ -n "$CROSS_BUILD" ]; then
+	hostmakedepends+=" python3-devel wayland-devel wayland-protocols libxkbcommon xkeyboard-config xkeyboard-config libxkbfile libxkbcommon-devel fontconfig-devel kitty"
```

Comment:
Please don’t exceed a line length of 80 characters for better readability. :)

Reply:
My editor spoils me for such things.....